DotNetHowToSecurityTrimming/DotNetHowToSecurityTrimming/Program.cs

```diff
@@ -0,0 +1,211 @@
+﻿#define HowToExample
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using Newtonsoft.Json;
+
+namespace DotNetHowToSecurityTrimming
+{
+    static class Program
+    {
+        public static string ClientId;
+
+        private static List<string> UsersPrincipalNameGroup;
```

Comment:
Why does it need to be a static field?

Reply:
Because it is being used within a static methods.

Reply:
Sorry, I meant, why does it need to be a field, instead of a local variable.